File: SIMPLEAPI_Demo/SIMPLEAPI_Demo/Clases/Empresa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SIMPLEAPI_Demo.Clases
{
    public class Contribuyente
    {
        public string RutEmpresa { get; set; }
        public string RazonSocial { get; set; }
        public string Giro { get; set; }
        public string Direccion { get; set; }
        public string Comuna { get; set; }
        public List<ActividadEconomica> CodigosActividades { get; set; }
        public DateTime FechaResolucion { get; set; }
        public int NumeroResolucion { get; set; }

        public int RutCuerpo { get { return int.Parse(RutEmpresa.Substring(0, RutEmpresa.Length - 2)); } }
        public string DV { get { return RutEmpresa.Substring(RutEmpresa.Length - 1, 1); } }

    }

    public class ActividadEconomica
    { 
        public int Codigo { get; set; }
    }
}

File: SIMPLEAPI_Demo/SIMPLEAPI_Demo_VB/Clases/Empresa.vb
Public Class Contribuyente
    Public Property RutEmpresa As String
    Public Property RazonSocial As String
    Public Property Giro As String
    Public Property Direccion As String
    Public Property Comuna As String
    Public Property CodigosActividades As List(Of ActividadEconomica)
    Public Property FechaResolucion As DateTime
    Public Property NumeroResolucion As Integer

    Public ReadOnly Property RutCuerpo As Integer
        Get
            Return Integer.Parse(RutEmpresa.Substring(0, RutEmpresa.Length - 2))
        End Get
    End Property

    Public ReadOnly Property DV As String
        Get
            Return RutEmpresa.Substring(RutEmpresa.Length - 1, 1)
        End Get
    End Property
End Class

Public Class ActividadEconomica
    Public Property Codigo As Integer
End Class
